ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -303,12 +303,6 @@ const PanelMain = fp.flow(
         analysisConfig.computeType === ComputeType.Dataproc,
         () => 'Reattachable disks are unsupported for this compute type',
       ],
-      [
-        runtimeExists &&
-          existingAnalysisConfig?.diskConfig?.detachable === false,
-        () =>
-          'To use a detachable disk, first delete your analysis environment',
-      ],
```

Comment:
Removing this condition as it will disable Reattachable persistent disk option in case the existing runtime is Standard storage disk. Which could lead to confusion as both the options will be disabled